kwave/kgrid.py

```diff
@@ -444,7 +440,7 @@ def highest_prime_factors(self, axisymmetric=None) -> np.ndarray:
                           largest_prime_factor(self.Nz)]
         return np.array(prime_facs)
 
-    def makeTime(self, c, cfl=CFL_DEFAULT, t_end=None):
+    def makeTime(self, c, cfl=0.3, t_end=None):
```

Comment:
Maybe assign it to `None` here and later re-assign it to `self.cfl_default` if not given? Otherwise, we will have two sources of truth.